### PR TITLE
Sync changelog with v2.8.1 release

### PR DESCRIPTION
**Pull request by Kami**
The title says it all.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -20,6 +20,19 @@ Added
...
+~~~~~~~
+
+Fixed
+~~~~~
+
+2.8.1 - July 18, 2018
+---------------------
+
+Added
+-----
```
Comment (blag) [inline CHANGELOG.rst]: Use `~~~~~` here?

Comment (Kami) [reply CHANGELOG.rst]: Yep, good catch, will fix it.

Comment (Kami) [reply CHANGELOG.rst]: @blag Pushed a fix.

File: contrib/runners/http_runner/http_runner/runner.yaml [hunk 2]
```diff
@@ -24,6 +24,7 @@
     password:
       description: Password required by basic authentication.
       type: string
+      secret: true
```
Comment (armab) [inline contrib/runners/http_runner/http_runner/runner.yaml]: The title still doesn't say it all :)

Comment (armab) [reply contrib/runners/http_runner/http_runner/runner.yaml]: As I understand we forgot to include this in master, while having the fix in `v2.8`

Comment (Kami) [reply contrib/runners/http_runner/http_runner/runner.yaml]: That's correct.

**Pull request merged**